src/main/resources/templates/fragments/selected-grouping.html

```diff
@@ -10,41 +10,43 @@
                 ng-click="returnToManagePerson(); cancelDescriptionEdit(); toggleShowAdminTab();">
             <i class="fas fa-arrow-left mr-1"></i> Return to Manage Person
         </button>
+
         <!-- Export to CSV button -->
-        <div class="btn-group" id="csvButton">
-            <div class="d-flex" tooltip data-original-title="{{ getCSVToolTipMessage() }}">
-                <button type="button" class="btn btn-primary btn-dropdown-split"
-                        ng-disabled="!paginatingComplete || groupingMembers.length === 0"
-                        ng-click="exportGroupToCsv(groupingMembers, 'members')">
-                    <i class="fa fa-users mr-1"></i>
-                    Export Members
-                </button>
-            </div>
-            <button type="button" class="btn btn-primary dropdown-toggle-split dropdown-toggle" data-toggle="dropdown"
-                    aria-label="More export options" aria-haspopup="true" aria-expanded="false"
-                    ng-disabled="!paginatingComplete || (groupingMembers.length === 0 && groupingExclude.length === 0)">
+        <div class="btn-group dropdown" id="csvButton" tooltip data-original-title="{{ getCSVToolTipMessage() }}">
+            <button type="button" class="form-control" data-toggle="dropdown"
+                    aria-haspopup="true" aria-expanded="false"
+                    ng-disabled="!paginatingComplete || (groupingMembers.length === 0 && groupingBasis.length === 0 && groupingInclude.length === 0 && groupingExclude.length === 0)">
+                <i class="fa fa-file-export mr-1"></i>
+                Export Grouping
```

Comment:
I like the new design of the dropdown button but I also think adding a caret/chevron would make the design even better because it tells the user that this is a dropdown. Here are some designs with a caret or a chevron:

Caret:
![image](https://github.com/uhawaii-system-its-ti-iam/uh-groupings-ui/assets/42422209/9eaeb31e-3581-4827-973a-b43167836b87)
I used this code after the "Export Grouping" text: `<i class="fa fa-caret-down ml-1"></i>`

Chevron:
![image](https://github.com/uhawaii-system-its-ti-iam/uh-groupings-ui/assets/42422209/a71b2e32-136d-4c72-9fa0-d1aced3fdd65)
I used this code after the "Export Grouping" text: `<i class="fa fa-chevron-down ml-1"></i>`

Let me know what you think.

Reply:
I lean more towards the caret, the chevron is used on the site for toggling the alphabetical order of names in the tables so using the caret would differentiate between the two functionalities. Though I personally find the chevron more aesthetically pleasing

Reply:
Ok let's go with the caret.